index.html

```diff
@@ -0,0 +1,944 @@
+                <span>Lorem ipsum.</span>&nbsp;
+                <span>Lorem ipsum dolor.</span>
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="article">
+          <div class="story flex">
+            <a class="banner" href=""></a>
+            <div class="text">
+              <a href="">
+                <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</h4>
+              </a>
+              <p>
+                <span>Lorem ipsum.</span>&nbsp;
+                <span>Lorem ipsum dolor.</span>
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="article">
+          <div class="story flex">
+            <a class="banner" href=""></a>
+            <div class="text">
+              <a href="">
+                <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</h4>
+              </a>
+              <p>
+                <span>Lorem ipsum.</span>&nbsp;
+                <span>Lorem ipsum dolor.</span>
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="article">
+          <div class="story flex">
+            <a class="banner" href=""></a>
+            <div class="text">
+              <a href="">
+                <h4>Lorem, ipsum dolor sit amet consectetur adipisicing elit.</h4>
+              </a>
+              <p>
+                <span>Lorem ipsum.</span>&nbsp;
+                <span>Lorem ipsum dolor.</span>
+              </p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </section>
+    <!--latest funding two-->
+    <section class="latest-funding container">
+      <div class="flex header-title">
+        <h2 class="title">Latest fund rounds on index.co ></h2>
+        <ul class="funding-links flex">
+          <li><a href="#">Add info</a></li>
+          <li><a href="#">Get Updates by email</a></li>
+          <li><a href="#">See more</a></li>
+        </ul>
+      </div>
+      <div class="latest-funding-cards">
+        <div class="latest-funding-card">
+          <div class="card-img">
+            <a href="#" class="card-header">
+              <img src="https://img-cdn.tnwcdn.com/image?url=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1213904702057922561%2FrnIhnMcE.jpg&amp;signature=56c88ec8cead593ebf039fc40c056866" alt="Sanzo">
+              <h4 class="card-title">Sanzo</h4>
+              <div class="card-data">
+                  <h4>$1.3M</h4>
+                  <h5>Seed fund</h5>
+              </div>
+            </a>
+            <div class="card-details">
+              <p><a href="#">Consumer Goods</a></p>
+              <p><a>Fisher Funds, +4</a></p>
+            </div>
+            <div class="more-card-details">
+              <p>Sanzo Raises $1.3M For the First Asian-Inspired Sparkling Water Brand</p>
+              <span>on AlleyWatch</span>
+            </div>
+          </div>
+        </div>
+        <div class="latest-funding-card">
```

Comment:
- On **desktop** you are now missing a "latest-funding-card". Maybe it disappeared when you did the previous changes. Please add it (only for desktop)
![image](https://user-images.githubusercontent.com/53256139/89997064-3fdaff80-dc51-11ea-852f-2a08d38bc881.png)